packages/core/__tests__/auth/__unit__/__snapshots__/Login.apiml.unit.test.ts.snap

```diff
@@ -8,7 +8,7 @@ REST API Failure with HTTP(S) status 401
 
 Host:      undefined
 Port:      undefined
-Base Path: undefined
+Base Path: 
```

Comment:
This doesn't seem quite right. If login fails with a 401, are we expecting the base path to be a space or similar, instead of undefined?

Reply:
In the ZosmfRestClient error handling, could we add a check for `this.session.ISession.tokenValue != null` on error messages for invalid token? This might make the base path trickery unnecessary 😋